dev/CommonStyles/TestUI/CornerRadiusPage.xaml

```diff
@@ -11,7 +11,7 @@
     mc:Ignorable="d"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
-    <ScrollViewer>
+    <controls:ScrollViewer CornerRadius="6">
```

Comment:
add control7Presenter? otherwise app may crash if user use the page on old os.

Reply:
This is in a CS test project, and it doesn't seem to support the UseNonstandardCondtionalXAML tag we are using in dev project. Is there a way to make it work? For now I just disabled the test on older os versions.